src/app/landing/LandingPage.jsx

```diff
@@ -57,34 +64,90 @@ class LandingPage extends Component {
               <p className="text-lighter">
-                GoGetta has New York’s most up-to-date service information on food, personal care, shelter, and clothing.
+                GoGetta has New York’s most up-to-date service information on
+                food, personal care, shelter, and clothing.
               </p>
             </div>
 
             <picture>
+              {/* desktop jpg */}
               <source
-                className="OnboardingImage"
-                media="(min-aspect-ratio: 2/3)"
-                srcSet="img/landing_page/map_tablet.png"
+                media="(min-width: 1224px)"
+                srcSet="/img/landing_page/step1_desktop1x.jpg 1x,
+                /img/landing_page/step1_desktop2x.jpg 2x"
+                type="image/jpeg"
+              />
+              {/* desktop webp */}
+              <source
+                media="(min-width: 1224px)"
+                srcSet="/img/landing_page/step1_desktop1x.webp 1x,
+              /img/landing_page/step1_desktop2x.webp 2x"
+                type="image/webp"
+              />
+              {/* mobile jpg */}
+              <source
+                media="(max-width: 1223px)"
+                srcSet="/img/landing_page/step1_mobile2x.jpg"
+                type="image/jpeg"
+              />
+              {/* mobile webp */}
+              <source
+                media="(max-width: 1223px)"
+                srcSet="/img/landing_page/step1_mobile2x.webp"
+                type="image/webp"
               />
               <img
+                src="/img/landing_page/step1_mobile2x.jpg"
                 className="OnboardingImage"
-                src="img/landing_page/map_narrow.png"
-                alt="map showing food, clothing, and personal care markers"
+                alt="map showing food, clothing, shelter, and personal care markers"
               />
             </picture>
           </div>
 
           <div className="Section">
             <div className="OnboardingSectionNumber">2</div>
-
             <div className="OnboardingText">
-              <h2>Find providers that will actually serve <span className="Emphasis">you</span></h2>
+              <h2>
+                Find providers that will actually serve{" "}
+                <span className="Emphasis">you</span>
+              </h2>
               <p className="text-lighter">
-                Filter or answer a few questions and we will show you locations that meet your specific needs, from age to gender to family situations.
+                Filter or answer a few questions and we will show you locations
+                that meet your specific needs, from age to gender to family
+                situations.
               </p>
             </div>
-            <img className="OnboardingImage" src="img/landing_page/welcome.jpg" alt="" />
+            <picture>
+              {/* desktop jpg */}
+              <source
+                media="(min-aspect-ratio: 2/3)"
```

Comment:
How come step 2 still uses ratio as the breakpoint, whereas 1 and 3 use width?

Reply:
Fixed